release_notes/Service_Fabric_ReleaseNotes_90CU3.md

```diff
@@ -0,0 +1,83 @@
+|Service Fabric PowerShell and CLI | AzureRM PowerShell Module  <br> SFCTL |  0.3.15  <br> 11.0.1 |
+
+
+## Contents 
+
+Microsoft Azure Service Fabric 9.0 Cumulative Update 3.0 Release Notes
+
+* [Breaking Changes](#breaking-changes)
+* [Key Announcements](#key-announcements)
+* [Service Fabric Feature and Bug Fixes](#service-fabric-feature-and-bug-fixes)
+* [Retirement and Deprecation Path Callouts](#retirement-and-deprecation-path-callouts)
+* [Repositories and Download Links](#repositories-and-download-links)
+
+## Breaking Changes
+Customers running Azure Service Fabric Linux Clusters will not be able to scale out or reimage (manual or auto via VMSS OS Image Update) nodes or create new Linux clusters without upgrading to this version of Service Fabric (9.0.1086.1). Also, if your application requires JDK, then you will need to install a JDK of your choice since SF will not install the JDK as part of the SF installation.
+Starting 8/19/2022, the following versions will be deprecated and will not be available on Linux for new cluster creation or node reimage/scale out operations:<br>
+* 9.0 CU2/CU1/RTO
+* 8.2 CU4/CU3/CU2.1/CU2/CU1/RTO<br>
+
+**Details:** Service Fabric Runtime (for Linux) placed a dependency on Zulu for Azure build of OpenJDK which is no longer supported as announced in their dev [blog](https://devblogs.microsoft.com/java/end-of-updates-support-and-availability-of-zulu-for-azure/). Since the creation of new Linux clusters or scale-out/reimage of nodes required all the dependencies to be installed, these workflows broke when the dependency became unavailable.<br>
+Beginning with this release, Service Fabric Runtime will no longer install any JDK dependency which will ensure that customer can scale out, reimage, or create new Linux clusters. If your application needs JDK, please utilize an alternate mechanism to provision a JDK. To get the latest Service Fabric Runtime releases with this change, follow the upgrade documentation. If you depend on a JDK, please be sure to install one of your choice.<br>
+For more information see: [Breaking change for Azure Service Fabric Linux customers](https://techcommunity.microsoft.com/t5/azure-service-fabric-blog/breaking-change-for-azure-service-fabric-linux-customers/ba-p/3604678)
+
+## Key Announcements
+* SFRP node types with VMSS Durability of ”Silver”/”Gold” should always have at least 5 VMs. If you need to deploy with less than 5 VMs, please consider using Durability "Bronze", but this is not recommended for Production clusters. This will be blocked in SFRP starting *09/30/2022*, and will start failing the deployments when the target VM count is less than 5.Initially, this will be enforced on new clusters, but gradually this validation will be enabled for all clusters. <br> For details see: [Durability characteristics of the cluster] (https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-cluster-capacity#durability-characteristics-of-the-cluster). 
```

Comment:
"This will be blocked in SFRP. . ." - what would be blocked? Production deployments with < 5 VMs? Or "Bronze" durability? Please clarify. Also, if "it" will be blocked, then why say it is supported but not "recommended".

Reply:
I feel we need to make this whole announcement clearer and more direct. e.g. If you need to deploy with less than 5 VMs, you must use Bronze Durability.

Our documentation states the following, we should keep the messaging aligned
"Production workloads require a durability level of Silver or Gold to help avoid data loss from VM-level infrastructure requests."